src/Views/TryInstallView.vala

```diff
@@ -0,0 +1,99 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+public class TryInstallView : AbstractInstallerView {
+    public signal void next_step ();
+
+    // replace this with actual content when the type is ready.
+
+    construct {
+        var title_label = new Gtk.Label (_("Install OS"));
+        title_label.get_style_context ().add_class ("h1");
+        title_label.valign = Gtk.Align.START;
+
+        var description_label = new Gtk.Label (_("You can install OS on this device now, or cancel the installation to try it without installing. "));
+        description_label.valign = Gtk.Align.START;
+        description_label.hexpand = true;
+
+        var nochanges_label = new Gtk.Label (_("Data from your previous operating system is unchanged until you install OS"));
+        nochanges_label.valign = Gtk.Align.START;
+        nochanges_label.wrap = true;
+
+        var nochanges_image = new Gtk.Image.from_icon_name ("computer", Gtk.IconSize.DIALOG);
+        nochanges_image.valign = Gtk.Align.START;
+        var nochanges_emblem = new Gtk.Image.from_icon_name ("emblem-default", Gtk.IconSize.LARGE_TOOLBAR);
+        nochanges_emblem.valign = Gtk.Align.END;
+        nochanges_emblem.halign = Gtk.Align.END;
+
+        var nochanges_overlay = new Gtk.Overlay ();
+        nochanges_overlay.add_overlay (nochanges_image);
+        nochanges_overlay.add_overlay (nochanges_emblem);
+        nochanges_overlay.halign = Gtk.Align.CENTER;
+        nochanges_overlay.width_request = 48;
+
+        var nosaving_label = new Gtk.Label (_("Any changes you make in OS before installing will not be saved"));
+        nosaving_label.valign = Gtk.Align.START;
+        nosaving_label.wrap = true;
+
+        var nosaving_image = new Gtk.Image.from_icon_name ("document-revert", Gtk.IconSize.DIALOG);
+        nosaving_image.valign = Gtk.Align.START;
+
+        var return_label = new Gtk.Label (_("If you cancel, you can always return to the installer by selecting the Install OS icon"));
+        return_label.valign = Gtk.Align.START;
+        return_label.wrap = true;
+
+        var return_image = new Gtk.Image.from_icon_name ("system-os-installer", Gtk.IconSize.DIALOG);
+        return_image.valign = Gtk.Align.END;
+
+        var help_grid = new Gtk.Grid ();
+        help_grid.orientation = Gtk.Orientation.VERTICAL;
+        help_grid.column_homogeneous = true;
+        help_grid.vexpand = true;
+        help_grid.margin_end = 60;
```

Comment:
No magic numbers. If the design calls for floating this grid center, then you should halign center.